files/service_files/disable_thp.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Disables THP on boot
+
+[Service]
+Type=oneshot
+ExecStart=/bin/hugeadm --thp-never
```

Comment:
Iv'e the feeling that this path isn't the same on all operating systems, but I really don't know.

Reply:
Correct, it's [`/usr/bin/hugeadm`](https://packages.debian.org/buster/amd64/libhugetlbfs-bin/filelist) on Debian, and it's not Arch where `/bin` is symlinked.